src/hotspot/share/gc/shenandoah/shenandoahHeap.inline.hpp

```diff
@@ -401,7 +401,7 @@ inline void ShenandoahHeap::marked_object_iterate(ShenandoahHeapRegion* region,
   ShenandoahMarkingContext* const ctx = complete_marking_context();
   assert(ctx->is_complete(), "sanity");
 
-  MarkBitMap* mark_bit_map = ctx->mark_bit_map();
+  const ShenandoahMarkBitMap* mark_bit_map = ctx->mark_bit_map();
```

Comment:
Why `const`? Not necessarily wrong, but inconsistent with the rest of the method.

Reply:
mark_bit_map() only returns a const now, and I think that's better: *if* we are to expose the bitmap, then better make it read-only.

Reply:
OTOH, it is only used to call get_next_marked_addr() which we can do just as well through ctx, and not expose the bitmap to begin with.